docs/reference/api-v3.yml

```diff
@@ -58,7 +81,7 @@ paths:
         description: Barcode of the product
     patch:
       summary: WRITE Product - Create or update product (API V3 - Implementation in progress)
-      operationId: post-api-v3-product-barcode
+      operationId: patch-api-v3-product-barcode
       responses:
         '200':
           description: |-
```

Comment:
Why is the explanation below not in the content schema with an entry "fields" ?

Reply:
There is a fields entry in the content schema for the request (which is above), below is the content schema of the response (which does not have a fields entry, but is affected by the fields parameter in the query)